### PR TITLE
feat(design): update DaffCardComponent

### PR DESCRIPTION
**Pull request by xelaint**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Part of: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: .stylelintignore [hunk 1]
```diff
@@ -0,0 +1,105 @@
+{
```
Comment (damienwebdev) [inline .stylelintignore:1]: Remove this file.

File: libs/design/src/molecules/card/card-media/card-media.directive.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import {
+  Directive,
+  HostBinding,
+} from '@angular/core';
+
+@Directive({
+  selector: '[daffCardMedia]',
```
Comment (damienwebdev) [inline libs/design/src/molecules/card/card-media/card-media.directive.ts:7]: I don't think this makes sense. I don't know of any use-case for supporting rendering a video inside a card. Seems likely that the UI (on a mobile device) is much too small. I think it makes more sense to keep Image until we have an explicit use-case for media.

File: libs/design/card/examples/src/examples.ts [hunk 3]
```diff
@@ -1,7 +1,9 @@
-import { CardWithColorComponent } from './card-with-color/card-with-color.component';
-import { CardWithImageComponent } from './card-with-image/card-with-image.component';
+import { BasicCardComponent } from './basic-card/basic-card.component';
+import { CardThemingComponent } from './card-theming/card-theming.component';
+import { RaisedCardComponent } from './raised-card/raised-card.component';
 
 export const CARD_EXAMPLES = [
```
Comment (damienwebdev) [inline libs/design/card/examples/src/examples.ts:5]: See #1430

**Review by damienwebdev — CHANGES_REQUESTED** [review]

File: .stylelintignore (hunk 1 above)
Comment (xelaint) [reply .stylelintignore]: removed

File: libs/design/card/examples/src/examples.ts (hunk 3 above)
Comment (xelaint) [reply libs/design/card/examples/src/examples.ts:6]: I don't think image needs a separate example.

File: libs/design/src/molecules/card/card-media/card-media.directive.ts (hunk 2 above)
Comment (xelaint) [reply libs/design/src/molecules/card/card-media/card-media.directive.ts]: Reverted.

File: libs/design/card/examples/src/examples.ts (hunk 3 above)
Comment (xelaint) [reply libs/design/card/examples/src/examples.ts]: updated

**Comment by damienwebdev** [comment]
@xelaint can you revert the package-lock.json? Otherwise, LGTM.

**Comment by xelaint** [comment]
@damienwebdev reverted

**Review by damienwebdev — APPROVED** [review]
lgtm

**Pull request merged**